examples/node-exporter.yaml

```diff
@@ -69,6 +69,15 @@ spec:
           requests:
             cpu: 102m
             memory: 180Mi
+        securityContext:
```

Comment:
With the exception of the `frontend`, I'm tempted to leave the examples directory out of this change for now.

They're not officially a part of the managed infrastructure and I'd rather not risk breaking anything which our [docs](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/node_exporter) point to.